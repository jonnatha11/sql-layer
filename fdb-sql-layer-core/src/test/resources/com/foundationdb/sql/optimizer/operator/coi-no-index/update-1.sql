UPDATE customers SET name = 'Smyth' WHERE name = 'Smith'