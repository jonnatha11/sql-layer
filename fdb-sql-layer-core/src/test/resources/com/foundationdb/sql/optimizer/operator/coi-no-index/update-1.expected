PhysicalUpdate
  Update_Returning(customers SET name = 'Smyth')
    Filter_Default(customers)
      Select_HKeyOrdered(customers.name == 'Smith')
        GroupScan_Default(customers)